src/pocketmine/entity/Living.php

```diff
@@ -355,18 +355,32 @@ protected function sendEffectRemove(EffectInstance $effect) : void{
 
 	}
 
+	/**
+	 * Returns the possibility to consume or not the given Consumable object.
+	 */
+	public function canConsumeObject(Consumable $consumable) : bool{
+		return $consumable instanceof MaybeConsumable and $consumable->canBeConsumed();
+	}
+
+	/**
+	 * Called when an Consumable object is going to be consumed.
```

Comment:
`is going to be` seems inappropriate here. This function is applying the results of consuming an item.